app/javascript/components/recipes/new/add-button.vue

```diff
@@ -0,0 +1,18 @@
+ <template> 
+  <button
+      @click="clickButton"
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 4 spaces but found 6 spaces                               vue/html-indent